edx_proctoring/views.py

```diff
@@ -779,3 +780,30 @@ def get(self, request):
             user_id=request.data.get('user_id', None),
             course_id=request.data.get('course_id', None)
         ))
+
+
+class ProctoredExamAttemptReviewStatus(AuthenticatedAPIView):
+    """
+    Endpoint for the Active Exams for a user.
```

Comment:
update the docstring. may be something `Endpoint for updating exam attempt review status to acknowledged`